lib/plugins/create/templates/aws-nodejs-ecma-script/serverless.yml

```diff
@@ -0,0 +1,26 @@
+service: babel-dynamically-entries-example
+
+# Add the serverless-webpack plugin
+plugins:
+  - serverless-webpack
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+
+functions:
+# Example without LAMBDA-PROXY integration
+# Invoking locally:
+# sls webpack invoke -f first
```

Comment:
@HyperBrain 
After serverless-webpack v3 maybe we could just remove this comment.